tools/fetch_aips

```diff
@@ -0,0 +1,140 @@
+#!/usr/bin/env python3
+import logging
+import os
+import sys
+from datetime import datetime
+
+import click
+from app import init
+from helpers import fetch
+
+from AIPscan import db
+from AIPscan.Aggregator import database_helpers
+from AIPscan.Aggregator.task_helpers import create_numbered_subdirs
+from AIPscan.models import StorageService
+from config import CONFIGS
+
+
+@click.command()
+@click.option("--ss-id", "-s", required=True, help="Storage service ID", type=int)
+@click.option("--session-id", "-i", required=True, help="Session descriptor", type=str)
+@click.option(
+    "--page", "-p", help="Page (if not set then all AIPs will be fetched)", type=int
+)
+@click.option(
+    "--packages-per-page",
+    "-n",
+    default=1000,
+    help="Packages per page (default 1000)",
+    type=int,
+)
+@click.option("--logfile", "-l", default=None, help="Filepath to log to", type=str)
+def main(ss_id, session_id, page, packages_per_page, logfile):
+    # Set up logging
+    logging.basicConfig(
+        level=logging.INFO,
+        format="%(asctime)s:%(levelname)s:%(name)s:%(message)s",
+        filename=logfile,
+    )
+
+    logger_name = os.path.basename(sys.argv[0])
+    logger = logging.getLogger(logger_name)
+    logger.setLevel(logging.INFO)
+
+    root = logging.getLogger()
+    root.setLevel(logging.INFO)
```

Comment:
At some point we should have a `get_logger` or similar for consistent logging across multiple tools.

Reply:
Sounds good! I've added an issue to add this.